libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -558,8 +560,8 @@ - (void) webView:(WKWebView *) webView decidePolicyForNavigationAction:(WKNaviga
              */
             [SFSDKWebUtils configureUserAgent:[self sfHybridViewUserAgentString]];
             [[SFAuthenticationManager sharedManager]
-             loginWithCompletion:^(SFOAuthInfo *authInfo) {
-                 
+             loginWithCompletion:^(SFOAuthInfo *authInfo,SFUserAccount *userAccount) {
+                 [SFUserAccountManager sharedInstance].currentUser = userAccount;
```

Comment:
This one's trickier to me.  If `currentUser` is _not_ equal to the returned user account after the hybrid refresh flow, what does that mean?  I feel like this is an area that's not siloed by user, where it should be.  Like maybe the view controller should have its own localized property describing its user?